Services/ContentBlocks/ContentBlockService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.ContentBlocks;
+
+namespace Andculture.Sitefinity.Testing.Services.ContentBlocks
+{
+    public class ContentBlockService : ODataService<ContentBlock>
+    {
+        public ContentBlockService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system/contentitems";
```

Comment:
Suggestion - Consider moving the path to a configuration store. That way we can quickly update it if it changes over time.